dts/bindings/device_node.yaml.template

```diff
@@ -61,4 +61,9 @@ properties:
   - cell1    # name of second cell
   - cell2    # name of third cell
   - and so on and so forth
+
+# In case "generation" category is set to "define", value "use-prop-name"
```

Comment:
The word 'category' is already used and means something else, I hope we can use a different word.

Reply:
Attribute maybe?

Reply:
ok, corrected.